EventListener/PsrResponseListener.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/*
+ * This file is part of the Symfony framework.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sensio\Bundle\FrameworkExtraBundle\EventListener;
+
+use Psr\Http\Message\ResponseInterface;
+use Symfony\Bridge\PsrHttpMessage\Factory\HttpFoundationFactory;
+use Symfony\Bridge\PsrHttpMessage\HttpFoundationFactoryInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\GetResponseForControllerResultEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+/**
+ * Converts PSR-7 Response to HttpFoundation Response using the bridge.
+ *
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+class PsrResponseListener implements EventSubscriberInterface
+{
+    /**
+     * @var HttpFoundationFactoryInterface
+     */
+    private $httpFoundationFactory;
+
+    public function __construct(HttpFoundationFactoryInterface $httpFoundationFactory = null)
+    {
+        if (null === $httpFoundationFactory) {
+            if (!class_exists('Symfony\Bridge\PsrHttpMessage\Factory\HttpFoundationFactory')) {
+                throw new \RuntimeException('Install the PSR-7 bridge to return a Psr\Http\Message\ResponseInterface instance in the controller.');
+            }
```

Comment:
Hm indeed, you're right. But shouldn't we provide a more meaningful message then if the user enables PSR-7 support while the bridge is not installed (in the `Configuration` class)?